judge/build_executable.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
+
+# Script to build executables.
+#
+# Usage: $0 <workdir>
+
+set -e
+trap 'cleanup ; error' EXIT
+
+cleanup ()
+{
+	$DJ_LIBJUDGEDIR/chroot-startstop.sh stop
+
+	# Make sure that all files are owned by the current user/group, so
+	# that we can delete the judging output tree without root access.
+	# We also remove group RUNGROUP so that this can safely be shared
+	# across multiple judgedaemons, and remove write permissions.
+	$GAINROOT chown -R "$(id -un):" $WORKDIR
+	chmod -R go-w $WORKDIR
+#	for i in bin etc lib lib64 proc usr; do
+#		rm -rf $i
+#	done
```

Comment:
This commented out cleanup code from me can be removed once we change the directory structure, see below.